<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>228</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string>Control De Inventarios Con Python Qt</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="oioio">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>ID</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="txtID"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="oioio_2">
           <property name="text">
            <string>Producto</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="txtProducto"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="oioio_3">
           <property name="text">
            <string>Cantidad</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="txtCantidad"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="oioio_4">
           <property name="text">
            <string>Precio</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="txtPrecio"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="btnAgregar">
           <property name="text">
            <string>Agregar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnBuscar">
           <property name="text">
            <string>Buscar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnEliminar">
           <property name="text">
            <string>Eliminar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLimpiar">
           <property name="text">
            <string>Limpiar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnExportar">
           <property name="text">
            <string>Exportar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QTableWidget" name="tabla">
           <column>
            <property name="text">
             <string>ID</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>PRODUCTO</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>CANTIDAD</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>PRECIO</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>510</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
